gammapy/cube/background.py

```diff
@@ -22,38 +22,38 @@ def make_map_background_irf(pointing, ontime, bkg, geom, n_integration_bins=1):
         Background rate model
     geom : `~gammapy.maps.WcsGeom`
         Reference geometry
-    n_integration_bins : int
-        Number of bins per energy bin in integration
+    integrate : bool
+        Whether to integrate the background model over the energy axes.
```

Comment:
I am not sure we should remove completely the possibility to have better integration but this is probably not the right place to pass the argument. We have two options: 
- we leave it to the user to have sufficient e_reco bins in the geometry
- we perform the proper integral using the bkg nodes
I tend to favor option 2. But in the meantime, let's go with option 1.